pkg/template/api/v1/types.go

```diff
@@ -63,4 +63,8 @@ type Parameter struct {
 
 	// Optional: Indicates the parameter must have a value.  Defaults to false.
 	Required bool `json:"required,omitempty" description:"indicates the parameter must have a non-empty value or be generated"`
+
+	// Optional: Indicates clients should expect the user to potentially provide a multiline value for this parameter value.
+	// Defaults to false
+	Multiline bool `json:"multiline,omitempty"`
```

Comment:
I am sure we will be asked for others at some point, we've already been
asked for file picker.  Number (float or int) is another typical one.

On Mon, Feb 29, 2016 at 8:06 AM, David Eads <notifications@github.com>
wrote:

> In pkg/template/api/v1/types.go
> <https://github.com/openshift/origin/pull/7673#discussion_r54406631>:
>
> > @@ -63,4 +63,8 @@ type Parameter struct {
> >
> >  	// Optional: Indicates the parameter must have a value.  Defaults to false.
> >  	Required bool `json:"required,omitempty" description:"indicates the parameter must have a non-empty value or be generated"`
> > +
> > +	// Optional: Indicates clients should expect the user to potentially provide a multiline value for this parameter value.
> > +	// Defaults to false
> > +	Multiline bool `json:"multiline,omitempty"`
>
> How soon are we going to rev a v2 and how soon are we going to get more
> hints? If we think we're going to have a multiple hints, I think I might
> want to have them segregated as a substruct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7673/files#r54406631>.
>